src/pages/injector/globalStyles.js

```diff
@@ -14,97 +13,108 @@
       font: inherit;
       vertical-align: baseline;
     }
+
+    // additional class here for when we need more
+    // specificity to override page styles, but also
+    // so we don't have to duplicate overrides to both 
+    // light & dark theme classes
+    .pocket-extension {
+      *:after,
+      *:before {
+        all: unset;
+      }
+    }
```

Comment:
Most of the changes in this file are whitespace fixes. This is really the only new bit.

Reply:
do we want to add the other reset chunk under this new class? the `header, footer, section, etc`?

Reply:
Not right now, I don't think. If we do, we will have to go through all the components to update their styling. Here's what it looks like w/ it nested:

<img width="427" alt="Screen Shot 2021-10-19 at 3 38 19 PM" src="https://user-images.githubusercontent.com/2893463/137987313-261aecee-4cfb-4abc-9f1c-2bde5fe1691f.png">